goal_src/levels/village_common/oracle.gc

```diff
@@ -5,3 +5,811 @@
 ;; name in dgo: oracle
 ;; dgos: CIT, L1, VI1, VI2, VI3, VILLAGEP
 
+(define-extern *oracle-sg* skeleton-group)
+
+;; definition of type oracle
+(deftype oracle (process-taskable)
+  ((first-task     uint8   :offset-assert 380)
+   (second-task    uint8   :offset-assert 381)
+   (left-eye-cell  handle  :offset-assert 384)
+   (right-eye-cell handle  :offset-assert 392)
+   )
+  :heap-base #x120
+  :method-count-assert 53
+  :size-assert         #x190
+  :flag-assert         #x3501200190
+  )
+
+;; failed to figure out what this is:
+(let
+  ((v1-1
+    (new 'static 'skeleton-group
+     :art-group-name "oracle"
+     :bounds (new 'static 'vector :w 16384.0)
+     :version #x6
+     )
+    )
+   )
+  (set! (-> v1-1 jgeo) 0)
+  (set! (-> v1-1 janim) 2)
+  (set! (-> v1-1 mgeo 0) (the-as uint 1))
+  (set! (-> v1-1 lod-dist 0) 4095996000.0)
+  (set! *oracle-sg* v1-1)
+  )
+
+;; definition for method 32 of type oracle
+(defmethod play-anim! oracle ((obj oracle) (arg0 symbol))
+  (with-pp
+   (set! (-> obj talk-message) (the-as uint 260))
+   (case (current-status (-> obj tasks)) 
```

Comment:
don't worry about it for this PR, but I think `current-status` should return a `task-status`, which should make some of these `case` statements less mysterious